.github/workflows/scripts/linux_arm64/test_wheels.sh

```diff
@@ -0,0 +1,31 @@
+set -e -x
+cd /catalyst 
+# Process args
+export GCC_VERSION=$1
+export PYTHON_VERSION=$2
+export PYTHON_SUBVERSION=$3
+export PYTHON_PACKAGE_NAME=$4
+# Install system dependencies
+dnf update -y 
+dnf install -y libzstd-devel gcc-toolset-${GCC_VERSION} ${PYTHON_PACKAGE_NAME} ${PYTHON_PACKAGE_NAME}-devel gcc
+dnf clean all -y 
+# Make GCC the default compiler
+source /opt/rh/gcc-toolset-${GCC_VERSION}/enable -y 
+export C_COMPILER=/opt/rh/gcc-toolset-${GCC_VERSION}/root/usr/bin/gcc 
+export CXX_COMPILER=/opt/rh/gcc-toolset-${GCC_VERSION}/root/usr/bin/g++ 
+# Set the right Python interpreter
+rm -rf /usr/bin/python3
+ln -s /opt/_internal/cpython-${PYTHON_VERSION}.${PYTHON_SUBVERSION}/bin/python3 /usr/bin/python3 
+export PYTHON=/usr/bin/python3
+# Set llvm-symbolizer
+ls -la /catalyst/llvm-build/bin/llvm-symbolizer
+export LLVM_SYMBOLIZER_PATH=/catalyst/llvm-build/bin/llvm-symbolizer
+# Add LLVM, Python and GCC to the PATH env var
+export PATH=/catalyst/llvm-build/bin:/opt/_internal/cpython-${PYTHON_VERSION}.${PYTHON_SUBVERSION}/bin:/opt/rh/gcc-toolset-${GCC_VERSION}/root/usr/bin:$PATH
+/usr/bin/python3 -m pip install numpy pybind11 PyYAML cmake ninja pytest pytest-xdist 'autoray<0.6.10' oqc-qcaas-client PennyLane-Lightning-Kokkos 'amazon-braket-pennylane-plugin>1.27.1'
```

Comment:
```suggestion
/usr/bin/python3 -m pip install numpy pybind11 PyYAML cmake ninja pytest pytest-xdist 'autoray<0.6.10' oqc-qcaas-client PennyLane-Lightning-Kokkos 'amazon-braket-pennylane-plugin>1.27.1'
```
```suggestion
/usr/bin/python3 -m pip install numpy pybind11 PyYAML cmake ninja pytest pytest-xdist autoray oqc-qcaas-client PennyLane-Lightning-Kokkos 'amazon-braket-pennylane-plugin>1.27.1'
```
Autoray should no longer be pinned.

Reply:
Done